backend/src/main/java/gov/cdc/usds/simplereport/validators/CsvValidatorUtils.java

```diff
@@ -292,9 +313,11 @@ public static MappingIterator<Map<String, String>> getIteratorForCsv(InputStream
   public static String convertEthnicityToDatabaseValue(String ethnicity) {
     Map<String, String> displayValueToDatabaseValue =
         Map.ofEntries(
-            Map.entry("hispanic or latino", "hispanic"),
-            Map.entry("not hispanic or latino", "not_hispanic"),
-            Map.entry("unk", UNKNOWN_LITERAL),
+            Map.entry(HISPANIC_LITERAL, HISPANIC_DB_VALUE),
+            Map.entry(HISPANIC_CODE, HISPANIC_DB_VALUE),
+            Map.entry(NOT_HISPANIC_LITERAL, NOT_HISPANIC_DB_VALUE),
+            Map.entry(NOT_HISPANIC_CODE, NOT_HISPANIC_DB_VALUE),
+            Map.entry(UNKNOWN_CODE, UNKNOWN_LITERAL),
             Map.entry(UNKNOWN_LITERAL, UNKNOWN_LITERAL));
```

Comment:
Similar to Race, but when I upload a [patient](https://dev2.simplereport.gov/app/patient/e4476dbd-0191-48d3-8849-5d036ac3eeb3?facility=1a0139b9-fc23-450b-9b6c-cd081e5cea9d) with "UNK" or "UNKOWN" ethnicity that section is empty. 
![image](https://user-images.githubusercontent.com/10108172/227981931-cad31a41-4e0c-4d7e-a76b-424898160396.png)